components/radio/index.css

```diff
@@ -447,9 +454,11 @@ governing permissions and limitations under the License.
     border-radius: 50%;
     border-width: var(--mod-radio-border-width,
                       var(--spectrum-radio-border-width));
-    border-color: var(--highcontrast-radio-gray-default,
+    /* border-color: var(--highcontrast-radio-gray-default,
                       var(--mod-radio-gray-default,
-                          var(--spectrum-radio-gray-default)));
+                          var(--spectrum-radio-gray-default))); */
+
+    border-color: var(--highcontrast-radio-button-border-color-default, var(--mod-radio-button-border-color-default, var(--spectrum-radio-button-border-color-default)));
```

Comment:
Commented out the old line, and put in a new line with the new custom properties that I wrote.